bootstrap/lib/aws.js

```diff
@@ -8,6 +8,8 @@ const {
   HTTP_ERROR_UNSUPPORTED_MEDIA_TYPE_MESSAGE,
```

Comment:
We should also add this feature to [bootstrap/lib/gcp.js](https://github.com/smartcontractkit/external-adapters-js/blob/master/bootstrap/lib/gcp.js)